taskiq/result_backends/redis/redis_backend.py

```diff
@@ -0,0 +1,90 @@
+import pickle
+from typing import Any, Dict, TypeVar
+
+from redis.asyncio import ConnectionPool, Redis
+from taskiq.abc.result_backend import TaskiqResult
+
+from taskiq import AsyncResultBackend
+
+_ReturnType = TypeVar("_ReturnType")
+
+
+class RedisAsyncResultBackend(AsyncResultBackend[_ReturnType]):
+    """Async result based on redis."""
+
+    def __init__(self, redis_url: str):
+        self.redis_url = redis_url
+
+    async def startup(self) -> None:
+        """Makes redis connection on startup."""
+        self.redis_pool = ConnectionPool.from_url(self.redis_url)
```

Comment:
Place this thing in `__init__`, because startup isn't called by default from client side. This function is intended to be called by workers to prepare data. For example, to create queues inrabbitmq.

Reply:
And also creating pool is not an async operation. It can be easily placed in `__init__`.